dash/include/dash/util/UnitLocality.h

```diff
@@ -63,12 +63,12 @@ class UnitLocality
   }
 
   UnitLocality(
-    dart_unit_t   unit)
-  : UnitLocality(dash::Team::All(), unit)
+    global_unit_t   unit)
+  : UnitLocality(dash::Team::All(), local_unit_t(unit))
   { }
```

Comment:
The idea here is to enforce the use of a global ID if the team is guaranteed to be `dash::Team::All()`.